### ISSUE TITLE
Use of Ctrl+C to copy text can cause accidental deregister of the client

### ISSUE THREAD

**Issue by maclobdell** [ARMmbed/mbed-os-example-pelion#107]
Background: 
For Windows users, it is common to use Ctrl+C to copy text.  If someone uses this command to try to copy text from the serial terminal window (example: coping the device ID), then it causes the client to deregister.  This can be confusing.  

Suggestion: 
Use a less command command, such as Ctrl+X to trigger the deregister.  

I have tested this, and it works.  
```
        } else if (in_char > 0 && in_char != 0x18) { // Ctrl+X is 0x18
            button_press(); // Simulate button press
            continue;
        }
        // if Ctrl+X, then reach this point and trigger deregister
        deregister_client();
        break;
```

**Comment by ciarmcom** [comment]
*Internal Jira reference: https://jira.arm.com/browse/IOTPART-9036*

**Comment by MarceloSalazar** [comment]
@maclobdell thanks for sharing this - I agree with the use case and proposal to change to Ctrl+X

**Comment by JanneKiiskila** [comment]
Yes, have to agree. It's a quite common use-case to use the copy-paste, so this makes perfectly sense.

**Comment by JanneKiiskila** [comment]
@maclobdell  - if you've already tested it, you could do a PR and let's get it merged in. Seems you have the code changes in your work area is, that way credit goes where it's due. :-)
